test/integration/processing/settlement/get-last-settlement.test.js

```diff
@@ -0,0 +1,101 @@
+const db = require('../../../../app/data')
+
+const { getLastSettlement } = require('../../../../app/processing/settlement')
+const { SETTLEMENT } = require('../../../mock-components/mock-dates')
+let currentSettlement
+let previousSettlement
+
+describe('process settlement', () => {
+  beforeAll(async () => {
+    await db.sequelize.truncate({
+      cascade: true,
+      restartIdentity: true
+    })
+  })
+
+  beforeEach(async () => {
+    currentSettlement = JSON.parse(JSON.stringify(require('../../../mock-settlement')))
+    currentSettlement.settlementDate = new Date(SETTLEMENT.DATE)
```

Comment:
This is like `new Date` inception ahaha, `SETTLEMENT.DATE` is `new Date(2022, 1, 8)`

so this is: `new Date(new Date(2022, 1, 8))`

comment above will resolve this